roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -41,6 +84,23 @@
     namespace: "{{ meta.namespace }}"
     conditions:
     - type: Available
-      status: "{{ 'True' if new_templates <= deployed_templates_after else 'False' }}"
+      status: "{{ 'True' if available else 'False' }}"
       reason: "available"
-      message: "Common templates available."
+      message: "Common templates available (deployed templates: {{ deployed_templates_after }}, desired deployed templated: {{ new_templates }})."
+
+- name: Set degraded condition
+  operator_sdk.util.k8s_status:
+    api_version: ssp.kubevirt.io/v1
+    kind: KubevirtCommonTemplatesBundle
+    name: "{{ meta.name }}"
+    namespace: "{{ meta.namespace }}"
+    conditions:
+    - type: Degraded
+      status: "{{ 'True' if degraded else 'False' }}"
+      reason: "degraded"
+      message: "Templates degraded (deployed templates: {{ deployed_templates_after }}, desired deployed templated: {{ new_templates }})."
+
+- name: Reque if KubevirtCommonTemplatesBundle not available
+  fail:
+    msg: "Requeing until KubevirtTemplateValidator is available"
```

Comment:
until all templates are available?

But that can never happen in fact, templates are not "active", so once you get here you have them all.

Reply:
Correct, will remove this